### DISCUSSION TITLE
How to limit the caching of packages on the uplink or cache specified packages instead of caching all packages

### DISCUSSION THREAD

**Discussion by yang-zhuo-wei** [verdaccio/verdaccio#4109] [Q&A] [answered]
I purchased a cloud server with low specifications, including a hard drive with only 30GB of space. I wanted to set up a private npm registry, but I found that Verdaccio caches all packages from upstream links. However, I only want to cache my private packages to ensure that my hard drive has enough space.

**Comment by juanpicado** [comment, marked as answer]
There is a property for the uplink named `cache` https://verdaccio.org/docs/uplinks , that avoid store tarballs, which take the most of the space.
